Marlin/Configuration_adv.h

```diff
@@ -804,4 +804,114 @@
 //#define EXPERIMENTAL_I2CBUS
 #define I2C_SLAVE_ADDRESS  0 // Set a value from 8 to 127 to act as a slave
 
+
+
+
+
+/**
+ *  
+ *  Spindle control
+ * 
+ *  The M3, M4 & M5 g-code commands are used to turn the spindle on and off & to set the
+ *  spindle speed.  In this implementation the M4 command is the same as the M3 (assumes
+ *  the spindle rotates in one direction only).
+ * 
+ *  SuperPid is a router/spindle speed controller used in the CNC milling community.
+ *  Marlin can be used to turn the spindle on and off.  It can also be used to set
+ *  the spindle speed from 5,000 to 30,000 RPM.  
+ *
+ *  You'll need to select a 3.3V - 5V signal for the ON/OFF function and optionally
+ *  choose a 0-5V PWM pin for the speed control.
+ *
+ *  uncomment the following to enable the ON/OFF function
+ */
+ 
+//#define SPINDLE_ENABLE
+ 
+/*
+ *
+ *  uncomment the following to enable speed control
+ */
+ 
+//#define SPINDLE_SPEED
+ 
+ /*
+ *
```

Comment:
Please pay attention to comment formatting. There's a space before the lines starting with asterisk, so the asterisks align, and the first line contains a "Doxygen-style" extra asterisk (for future use with Doxygen).
```cpp
/**
 *
 *
 *
 */
```